src/api/services/Overpass.ts

```diff
@@ -25,4 +26,30 @@ export default {
       { enabled: !!box, refetchOnWindowFocus: false },
     );
   },
+  useNearestPlaces: (cityName: string, coordinates?: string) => {
+    const query = `[out:json][timeout:25];
+    (
+      node["place"~"town|city"](around:50000,${coordinates});
+    );
+    out body 7;
+    >;
+    out skel qt;`;
+
+    return useQuery<Place[], Error>(
+      ['cities', cityName],
+      async () => {
+        const res = await overpassClient.post('/interpreter', query);
+
+        const normalizedNearestPlaces = res.data.elements
+          .map((city: Place) => {
+            return { ...city, tags: { name: convertSpecialCharacters(city.tags.name) } };
+          })
+          .filter((city: Place) => city.tags.name !== cityName)
+          .slice(0, 6);
```

Comment:
**question** I'm wondering if this is solvable but when we get 7 cities and there is no our city it will remove 1 city and it could be the biggest city around our city.

Reply:
Overpass API returns these 7 cities very randomly, so we also thought we could randomly delete one city. But if we decide that such sorting is needed, it won't be difficult to solve this problem.